css-view-transitions-1/Overview.bs

```diff
@@ -936,8 +936,8 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 		[Exposed=Window]
 		interface ViewTransition {
 			readonly attribute Promise<undefined> updateCallbackDone;
-			readonly attribute Promise<undefined> ready;
-			readonly attribute Promise<undefined> finished;
+			[SameObject] readonly attribute Promise<undefined> ready;
+			[SameObject] readonly attribute Promise<undefined> finished;
```

Comment:
This is not valid Web IDL, technically. https://github.com/whatwg/webidl/issues/71